.github/workflows/main.yml

```diff
@@ -0,0 +1,587 @@
+name: build_macos
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - '20[2-9][0-9]-[0-9][0-9]-[0-9][0-9]'
+  pull_request:
+    paths-ignore:
+      - '**.md'
+    branches:
+      - master
+
+jobs:
+  macos-deps-build:
+    name: '01 - Build macOS dependencies'
+    runs-on: [macos-11]
+    strategy:
+      matrix:
+        arch: ['x86_64', 'arm64']
+    env:
+      MACOSX_DEPLOYMENT_TARGET: '10.13'
+      CACHE_REVISION: '02'
+      LIBPNG_VERSION: '1.6.37'
+      LIBPNG_HASH: '505e70834d35383537b6491e7ae8641f1a4bed1876dbfe361201fc80868d88ca'
+      LIBOPUS_VERSION: '2021-05-12'
```

Comment:
I understand why it's appealing to set the version to 2021-05-12.  However, if you run the `update_version` script included in the xiph/opus repo with this commit checked out, it will output a `package_version` file with "1.3.1-93-gdfd6c88a" as its set package version.  It makes more sense to use the authoritative version info rather than making up our own.  We should either leave this at 1.3.1, or use the version info generated by their script.